frontend/src/components/generic/CategoryChip.vue

```diff
@@ -16,6 +16,7 @@
     </slot>
     <slot name="after" />
   </GenericChip>
+  <GenericChip v-else v-bind="$attrs" v-on="$listeners">·&#x202f;·&#x202f;·</GenericChip>
```

Comment:
I'd rather have a real loading sr-only text, instead of another unicode symbol. Horizontal ellipsis is more used to truncate words. My symbols are only used as a decoration.

I will add a sr-only text